lib/recipients/pil.js

```diff
@@ -8,6 +8,8 @@ module.exports = ({ schema, logger, task }) => {
   const { Profile } = schema;
   const applicantId = get(task, 'data.subject');
   const establishmentId = get(task, 'data.establishmentId');
+  const action = get(task, 'data.action');
+  const ignoredActions = ['create', 'delete'];
```

Comment:
I think this needs to also ignore update actions on non-granted PILs.

Reply:
This seems to no-longer be a thing. PILs are created and then granted, not updated.